src/generator/AutoRest.Ruby.Azure.Tests/RspecTests/paging_spec.rb

```diff
@@ -129,6 +129,30 @@
     expect(items.count).to eq(10)
   end
 
+  it 'should get multiple pages with fragmented next link - lazy' do
+    page = @client.paging.get_multiple_pages_fragment_next_link_as_lazy('1.6', 'test_user')

```

Comment:
I see 1.6 & test_user are used in atleast 2 cases and most probably in more in the future. Can we define them in one place and use it everywhere